src/lib/screens/IssueIntent.svelte

```
@@ -54,6 +52,66 @@
+      return "No single recipient is valid on every output chain type — use outputs from one chain type.";
+    }
+    return RECIPIENT_FORMAT_HINT[outputChainTypes[0] ?? "evm"];
+  });
+
+  // Solana fills batch all of a chain's outputs into ONE transaction — the fill
+  // reference stored per output must contain every output's OutputFilledEvent
+  // (see fillOutputs in src/lib/solana/writes.ts) — and two exclusive outputs
+  // already exceed Solana's 1232-byte transaction cap. Such an order opens,
+  // escrows the inputs, and then can never be filled, so it is blocked here.
+  const solanaOutputOverflow = $derived.by(() => {
+    const perChain = new Map<number, number>();
```

ESLint

[error] Found a mutable instance of the built-in Map class. Use SvelteMap instead.

(svelte/prefer-svelte-reactivity)